### PR TITLE
Finally fix filter settings not getting applied

### PR DESCRIPTION
**Pull request by X9VoiD**
There were two problems, the incomplete name when querying the PluginSetting dictionary, and the filter being reconstructed (yeah, it was actually just half-fixed)

Now the filters are only constructed once, instead of three times.

### CODE REVIEW

File: TabletDriverPlugin/Output/AbsoluteOutputMode.cs [hunk 1]
```diff
@@ -14,15 +14,17 @@ public abstract class AbsoluteOutputMode : BindingHandler, IOutputMode
...
             {
-                _filters = value;
-                _preFilters = value.Where(f => f.FilterStage == FilterStage.PreTranspose).ToList();
-                _postFilters = value.Where(f => f.FilterStage == FilterStage.PostTranspose).ToList();
+                _filters = value.ToList();
+                foreach (IFilter filter in _filters)
+                    if (filter.FilterStage == FilterStage.PreTranspose)
+                        _preFilters.Add(filter);
+                    else if (filter.FilterStage == FilterStage.PostTranspose)
+                        _postFilters.Add(filter);
```
Comment (InfinityGhost) [inline TabletDriverPlugin/Output/AbsoluteOutputMode.cs:27]: This change still seems entirely unnecessary. By doing this you need to clear the list every time the filters change or settings apply.

Comment (X9VoiD) [reply TabletDriverPlugin/Output/AbsoluteOutputMode.cs:27]: I forgot that with this change, _pre and _post will leak the previous _filters, then Add() the new ones.

Comment (X9VoiD) [reply TabletDriverPlugin/Output/AbsoluteOutputMode.cs]: Clearing and rebuilding pre and post every filter and settings change is lighter than doing it on the fly during Transpose()

is what I've come up with.

File: TabletDriverPlugin/Output/AbsoluteOutputMode.cs [hunk 2]
```diff
@@ -14,15 +14,19 @@ public abstract class AbsoluteOutputMode : BindingHandler, IOutputMode
...
-                _preFilters = value.Where(f => f.FilterStage == FilterStage.PreTranspose).ToList();
-                _postFilters = value.Where(f => f.FilterStage == FilterStage.PostTranspose).ToList();
+                _filters = value.ToList();
+                _preFilters.Clear();
+                _postFilters.Clear();
+                foreach (IFilter filter in _filters)
+                    if (filter.FilterStage == FilterStage.PreTranspose)
+                        _preFilters.Add(filter);
+                    else if (filter.FilterStage == FilterStage.PostTranspose)
+                        _postFilters.Add(filter);
```
Comment (InfinityGhost) [inline TabletDriverPlugin/Output/AbsoluteOutputMode.cs:29]: Apply this change to RelativeOutputMode and I'll merge

Comment (X9VoiD) [reply TabletDriverPlugin/Output/AbsoluteOutputMode.cs:29]: Applied

**Pull request merged**